scripts/app.js

```diff
@@ -8,18 +8,23 @@ function Project (properties) {
   this.location = properties.location;
 }
```

Comment:
I did that this morning so you should see it in the next submit! Thanks for your suggestion :)